### ISSUE TITLE
Resizing the PMC window splitter to left makes the list a mess

### ISSUE THREAD

**Issue by nkolev92** [NuGet/Home#8116]
[658461](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/658461)

Repro steps: 

1. Launch VS and create a C# Blank App (Universal Windows).

2. Open “Package Manager Console”.

3. Drag the PMC window splitter to right.

4. Type ‘Get-Package -Updates’ in PMC.

5. Drag the PMC window splitter to left and observe the content in PMC.

Expected:

The list should be wrapped intelligently.

 Actual:

Resizing the PMC window splitter to left makes the list a mess.

 

Notes:

1. The issue repro on D16.0stg 28002.3000 (default NuGet Client Release-4.8.0-preview3/4.8.0.5278).

2. The issue repro on D15.8 28002.3000 + NuGet Client Release-4.8.0-RTM\4.8.0.5374.

**Comment by ghost** [comment]
This issue will now be closed since it had been marked Icebox cleanup candidate, but received no further activity in the past 14 days.
